stylesheet/style.css

```diff
@@ -0,0 +1,118 @@
+body{
+  background-color: #EAEAEA;
+}
+.header-container{
+  display: flex;
+flex-direction: column;
+flex-wrap: wrap;
+}
+ li{
+  display: inline;
+  font-size: 1.2rem;
+}
+.logo-bar {
+  background-color: #093C60;
+}
+.image-bar img{
+  height: 200px;
+  flex:2;
+}
+.logo-bar img{
+  height: 200px;
+  flex:1;
+}
+.main-container{
+display: flex;
+background-color: white;
+}
+.dogs-container{
+  display: flex;
+flex-wrap: wrap;
+justify-content:space-between;
+background-color: white;
+}
+.blog-container{
+  display: flex;
+  flex-direction: row;
+justify-content:space-between;
+background-color: white;
+width:100%;
+}
+.main-container div, .dogs-container div , .blog-container div{
+  padding:10px;
+}
+.box-1
+{
+flex: -1;
+}
+.box-2
+{
+flex:1;
+}
+.box-3
+{
+flex:1;
+}
+.dog-container-box{
+  width:22%;
+  background-color: white;
+}
+.blog-container-box{
+  display: flex;
+ flex-direction: column;
+ flex: 1;
+}
+.blog-image-container-box{
+  flex-grow: -1;
+}
+.dog-card-img{
+  width: 100%;
+}
+.sidebar
+{
+  float: right;
+  width: 70%;
+  padding-left: 20px;
+}
+.btn{
+  padding: 8px 80px;
+  font-size: 18px;
+  border-radius: 4px;
+  color: white;
+  text-align: center;
+  background-color: #81B640;
+}
+.btn-adopt{
```

Comment:
this isn't quite right, when you go to smaller widths the button goes out of alignment...

this should help

```
    /* margin-left: 45px; */
    margin: 0 auto;
    display: block;


``